src/mono/wasm/templates/templates/browser/browser.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <TargetArchitecture>wasm</TargetArchitecture>
+    <TargetOS>Browser</TargetOS>
+    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    <UseMonoRuntime>true</UseMonoRuntime>
+    <WasmMainJSPath>main.js</WasmMainJSPath>
+    <OutputType>Exe</OutputType>
+    <WasmEnableES6>yes</WasmEnableES6>
```

```suggestion
    <WasmEnableES6>true</WasmEnableES6>
```

It should be `true` to take effect.

---

Wonder how that got in :-)